app/uk/gov/hmrc/exports/controllers/SubmissionController.scala

```diff
@@ -49,6 +49,13 @@ class SubmissionController @Inject() (authenticator: Authenticator, submissionSe
     }
   }
 
+  def findAction(actionId: String): Action[AnyContent] = authenticator.authorisedAction(parse.default) { _ =>
```

Comment:
We are not linking the lookup of an action ID with the EORI passed in the authenticated header. This could lead to leaking other people's actions to a particular users!

Can we do something similar to looking up a submission by its ID please?